liminal/core/config/defaults/base/liminal.yml

```diff
@@ -21,12 +21,24 @@ type: super
 executors:
   - executor: default_k8s
     type: kubernetes
+  - executor: default_executor
+    type: default
 service_defaults:
   description: add defaults parameters for all services
 task_defaults:
   description: add defaults parameters for all tasks separate by task type
   python:
     executor: default_k8s
+  spark:
+    executor: default_k8s
+  job_end:
```

Comment:
Would be better to change the code so that we don't need to add every task type here with default_executor but that default one will be used if no other was found.

Reply:
done